docs/how-to/rest-apis.txt

```diff
@@ -0,0 +1,155 @@
+.. _rest-api-getting-started:
+
+Getting Started with REST APIs
+==============================
+
+REST APIs give users and external tools a way to interact with Determined programmatically.
+We provide detailed documentation on all the available functionalities (aka endpoints) and
+alongside a playground to interact with them through Swagger UI.
+
+Swagger UI
+----------
+
+As Swagger puts it:
+
+Swagger UI allows anyone — be it your development team or your end consumers —
+to visualize and interact with the API’s resources without having any of the
```

Comment:
Is there some kind of quote box we can put this in to visually distinguish it from our text?

Reply:
it looks indenting it is the way to go https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks